js/scorm/wrapper.js

```diff
@@ -465,12 +478,33 @@ define([
       this.commit();
     }
 
-    handleError(_msg) {
-      this.logger.error(_msg);
+    handleError(error) {
+
+      if (!Adapt.get('_isStarted')) {
+        Adapt.once('contentObjectView:ready', this.handleError.bind(this, error));
+        return;
+      }
+
+      const config = Adapt.course.get('_spoor');
+      const errorMessages = Object.assign({}, ScormError.defaultMessages, config && config._errorMessages);
+      const message = Handlebars.compile(errorMessages[error.name])(error.data);
+
+      switch (error.name) {
+        case CLIENT_COULD_NOT_CONNECT:
```

Comment:
@moloko Not sure if you want to handle any other errors this way, or if you want to be able to configure?

Reply:
TBH 'could not connect' is the big one... the other one that *might* be useful is [this one](https://github.com/adaptlearning/adapt-contrib-spoor/blob/68eb444bac3926b69da56cb98061b142d3664fe6/js/scorm/wrapper.js#L254)

Reply:
... which I notice now you've already handled!

Reply:
'Course is not connected to the LMS' doesn't display as a notify, only 'Could not connect' is notified atm.

Reply:
that's absolutely fine

Reply:
I figured I'd do all the ones that end up with a user popup whilst I was there. Not sure how useful that is?

Reply:
I think it's only right and proper that Adapt, claiming to have multilanguage support, should be able to display error messages in the learner's choice of language!